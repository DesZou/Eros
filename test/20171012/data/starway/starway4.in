10000 30000 400
468 1815
6267 10372
3383 26287
6968 24189
668 29871
6987 134
4102 9683
721 11596
5930 14263
1524 22300
9461 16503
8128 16267
1604 17523
7285 9652
1695 23125
1135 2125
7356 20444
4100 29716
86 19871
3756 3730
6986 1387
7251 12952
4986 24616
3911 22482
3023 25508
6543 28430
2945 30
6895 26074
773 6142
1484 28470
4274 1788
9563 6424
5293 3939
7086 13375
7019 26765
4361 18671
9402 22808
6201 10998
5137 8882
6912 14633
7118 29067
3258 26215
1069 3718
2359 29964
5392 4432
4128 2142
2133 10618
9496 20553
4963 19999
6942 29445
5479 28365
7143 29579
6092 19131
8424 16774
2531 21809
6986 28319
8416 16253
6056 13956
9160 11438
5408 22844
6185 18386
7093 1470
6398 25705
1506 834
25 24550
4876 2007
7581 5793
2003 15384
1925 21190
2513 28117
8221 16308
372 21360
6630 23405
8602 23865
1830 3683
423 7270
2747 26751
8346 193
8657 4618
5605 7962
3574 2932
8054 4666
4177 4686
2734 22406
4358 13181
5635 5896
8614 22145
95 8529
7307 11880
633 8715
5503 5733
2811 24146
1905 16129
1273 24565
8108 29483
5376 8631
3952 21463
519 10074
8318 10838
8807 7326
7938 11889
3910 21128
5120 25857
4726 20762
4620 1163
6778 13838
2450 27670
5815 4842
3167 10857
1710 19033
8879 5994
2275 26836
3808 22795
3303 18783
7895 10324
751 27103
8930 28198
6098 18383
4705 17887
7827 16169
6207 11500
4564 14060
1896 21225
7652 6168
4757 8778
7735 12621
437 19363
4124 22540
5053 15478
1030 22108
2076 1114
1600 2261
6055 25792
5749 5862
3843 22286
3967 19975
4434 4517
2575 12402
5688 27507
9005 19275
716 7170
6436 16480
1641 24129
5869 19894
8933 22558
6408 9441
8923 11621
8749 23114
9120 22744
4129 25209
2859 6778
9247 27597
6437 2621
5365 23037
5329 6035
8460 23408
2279 8878
2508 3119
1122 9119
4387 7012
3278 3426
4495 7248
5769 1384
8482 23570
2966 6746
1030 8704
4143 2980
6845 28054
3745 28133
2154 8359
1434 28973
719 3327
6427 12508
3299 5925
9313 10547
4736 8515
94 16833
3615 13058
4564 24155
6985 10241
9128 7166
5291 26076
7707 21517
6577 1953
9014 26906
9827 26204
6145 3965
7367 5575
8522 18584
1623 18245
39 2813
5288 13566
6719 29731
8927 11165
9173 13428
5112 4871
6077 19392
9845 21155
9338 4320
5264 523
6954 23544
2172 12287
44 22587
3916 11867
9576 423
8449 1027
3843 7068
7801 9738
8592 8313
3928 11320
7033 9246
9630 20940
6718 348
4188 13258
490 3419
9291 16556
7861 25002
9631 6407
6407 29186
5713 21987
5490 16104
9564 703
4371 19409
5487 5510
8986 22419
6558 12541
172 9043
718 22759
6633 14779
6991 5784
9636 7221
8735 7352
9009 19087
9503 18090
4100 13427
4187 17634
9441 22029
8082 5051
5043 11688
6509 9819
2518 15754
6678 15858
4547 28972
5065 4952
6396 12984
8861 28474
2308 24928
1357 6531
2417 6103
3304 4288
8732 19142
9417 2084
2570 27275
9359 10410
1035 22702
5553 6704
8384 10408
3273 29019
8294 11803
1132 26775
6761 1351
1773 28745
9815 29583
355 4842
5311 11036
326 8210
6106 10616
603 29539
3233 56
244 28327
6908 4929
6431 16088
8256 6140
3631 21158
3801 12027
978 7033
2192 19347
104 18645
4174 27267
5837 10700
5137 23827
1773 24704
7545 26413
4448 5591
1152 11373
3208 17019
5653 13983
8434 5317
8027 11805
7306 11182
2170 19533
7383 16713
5500 10455
6983 21693
8611 2669
4322 1549
6695 10493
862 20169
9402 15046
5760 7209
1312 28608
6479 5210
996 762
1173 7585
3203 29540
3851 8742
584 2276
3043 23942
6019 10526
915 26815
462 24412
331 11405
8931 9352
7846 8075
3176 6327
1779 2353
7823 25564
5661 15402
2515 9475
2144 10332
7905 22800
5099 14258
9266 27300
1482 28706
5700 23803
2150 3287
3210 251
8183 12035
3686 10686
1228 17090
5176 3494
9705 12237
4758 130
9315 18125
1428 17314
8199 27686
7150 20361
6210 28771
8468 6630
224 12097
2668 21854
5562 12239
707 21766
51 22872
8315 19014
7062 19595
7695 26624
4640 24190
8720 29738
4876 7040
7249 16699
1430 26038
3186 12354
1192 4262
8240 24726
4341 15239
6665 19813
5931 10182
2475 21098
4249 16676
8335 24761
1617 2342
4047 1692
8754 10763
4547 19811
1256 28545
9977 19650
81 2883
6739 21786
8138 5891
222 4505
4424 23842
8987 13979
6672 8979
472 22701
1208 1436
6382 11728
1725 8052
9187 17358
4044 8576
208 20494
5373 6653
5589 691
9023 11660
9287 3177
6973 3660
1845 4556
913 25242
4380 20317
7878 4358
6300 29750
4360 27297
8819 10648
5841 2006
9577 10355
8052 26543
2530 28121
6425 5894
500 14958
9787 20363
