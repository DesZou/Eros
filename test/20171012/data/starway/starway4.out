564.475420191
